### PR TITLE
Run Dora Run

### PR DESCRIPTION
**Pull request by SarthakJain2**
Dora will move when clicked on

### CODE REVIEW

File: challenge/package.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
         "react-scripts": "^4.0.1"
     },
     "scripts": {
-        "start": "react-scripts start",
+        "start": "react-scripts --openssl-legacy-provider start",
```
Comment (GabrielWeinbrenner) [inline challenge/package.json]: what is this change for?

**Review by GabrielWeinbrenner — CHANGES_REQUESTED** [review]
Please remove .DS_Store and package-lock.json

File: challenge/package.json (hunk 1 above)
Comment (SarthakJain2) [reply challenge/package.json]: I believe during the interview I asked you and Serena about why my npm start was not working. You guys suggested to add this and it fixed the problem. I believe it had something to do with my node version.

Comment (GabrielWeinbrenner) [reply challenge/package.json]: Oh right, can you leave that out of the commit. That may cause errors for other people.

Comment (SarthakJain2) [reply challenge/package.json]: I added a new commit that removes the information that I added. I hope thats okay. Thank you.

**Pull request closed without merging**